src/OracleService/OracleService.cs

```diff
@@ -225,18 +229,8 @@ private static async Task SendContentAsync(Uri url, string content)
         {
             try
             {
-                HttpWebRequest request = (HttpWebRequest)WebRequest.Create(url);
-                request.Method = "POST";
-                request.ContentType = "application/json";
-                request.Timeout = 5000;
-                using (StreamWriter dataStream = new StreamWriter(await request.GetRequestStreamAsync()))
-                {
-                    await dataStream.WriteAsync(content);
-                }
-                HttpWebResponse response = (HttpWebResponse)await request.GetResponseAsync();
-                if (response.ContentLength > ushort.MaxValue) throw new Exception("The response it's bigger than allowed");
```

Comment:
We will lose this condition?

Reply:
But we don’t care about response at all, do we?

Reply:
Is the download time included in the timeout ?